tests/test_lab.py

```diff
@@ -160,7 +160,7 @@ def test_ilab_cli_help(command: Command, cli_runner: CliRunner):
     cmd = command.get_args("--help")
     assert "--help" in cmd
     result = cli_runner.invoke(lab.ilab, cmd)
-    assert result.exit_code == 0, result.stdout
+    assert result.exit_code == 2, result.stdout
```

Comment:
no we should just not test return code at all. otherwise this change makes tests fail on older click version. the return code is of no consequence.

Reply:
I'm ok with removing the exit code check. This test was bordering on testing "implementation not behavior"

Reply:
Right, I was wondering but wasn't sure. I think they only changed return code on "bare" call (when no "correct" option is passed - and `--help` is one of the correct options.)